engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/views/content-view-publish.html

```diff
@@ -13,8 +13,16 @@ <h3 translate>Version Details</h3>
 
 <div class="row">
   <div class="col-sm-5">
-    <form name="versionForm" role="form" novalidate>
+    <div bst-alert="info" ng-show="contentView.duplicate_repositories_to_publish.length > 0">
+      You have selected more than one component Content View Version with the same repository resulting in slower publishing:
+      <ul>
+        <li ng-repeat="repository in contentView.duplicate_repositories_to_publish">
+          {{ repository.name }}
```

Comment:
Can you also tell the use the offending content view versions that hold this duplicate CV?.

Reply:
something like ->  foo belongs to components MegaFoo-1.0 and SuperFoo-2.0

Reply:
Added:  

![image](https://user-images.githubusercontent.com/395077/33920264-fb630cc8-df8a-11e7-9700-59044ceab60e.png)

Reply:
is that clear enough?

Reply:
perfect :)